mutiny-core/src/nostr/mod.rs

```diff
@@ -1172,6 +1197,242 @@ impl<S: MutinyStorage> NostrManager<S> {
         Ok(event_id)
     }
 
+    /// Creates a recommendation event for a federation
+    pub async fn recommend_federation(
+        &self,
+        invite_code: &InviteCode,
+    ) -> Result<EventId, MutinyError> {
+        let kind = Kind::from(18173);
+        // first pull down our current recommendation event, we need to do this
+        // to make sure we don't overwrite any other recommendations
+        let filter = Filter::new().kind(kind).author(self.public_key).limit(1);
```

Comment:
it's a single replaceable event with a list of other recommendations? I fucking hate nostr lists, come on.

Reply:
spec is updated to be a single event per recommendation